engine/schema/src/main/resources/META-INF/db/schema-41310to41400.sql

```diff
@@ -19,6 +19,10 @@
 -- Schema upgrade from 4.13.1.0 to 4.14.0.0
 --;
 
+-- Update the description to indicate this only works with KVM + Ceph 
+-- (not implemented properly atm for KVM+NFS/local, and it accidentaly works with XS + NFS. Not applicable for VMware)
+UPDATE `cloud`.`configuration` SET `description`='Indicates whether to always backup primary storage snapshot to secondary storage. Applicable for KVM + Ceph only.' WHERE  `name`='snapshot.backup.to.secondary';
```

Comment:
nice to update in the db, but it needs updating in code: https://github.com/apache/cloudstack/blob/master/server/src/main/java/com/cloud/storage/snapshot/SnapshotManager.java#L59